features/mbedtls/platform/COMPONENT_PSA_SRV_IMPL/src/default_random_seed.cpp

```diff
@@ -5,7 +5,16 @@
 
 int mbed_default_seed_read(unsigned char *buf, size_t buf_len)
 {
-    psa_its_status_t rc = psa_its_get(MBED_RANDOM_SEED_ITS_UID, 0, buf_len, buf);
+    psa_its_status_t rc;
```

Comment:
Will `mbed_default_seed_read()` ever be called multiple times? Perhaps mbedtls will poll, asking for 32-byte chunks at a time. Do we need to handle that? If so, we should make sure we don't re-use bits of injected entropy for each call to `mbed_default_seed_read()`.

Alternatively, we can set `MBEDTLS_ENTROPY_MIN_HARDWARE` to exactly how much was injected and exactly how much entropy Mbed TLS will ask for, and can return an error if the function is called more than once.